javascript/javascript1/week3/tvTime.js

```diff
@@ -0,0 +1,32 @@
+const seriesDurations = [{
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+},
+{
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+},
+{
+    title: "The Wire",
+    days: 6,
+    hours: 17,
+    minutes: 0,
+}
+];
+let totalTime = 0;
+const totalLifeTime = 80 * 365 * 24 * 60;
+
+function logOutSeriesText(seriesDurations) {
+for (let i = 0; i < seriesDurations.length; i++) {
+    let SeriesTimeDurationInMinutes = (seriesDurations[i].days * 24 * 60) + (seriesDurations[i].hours * 60) + (seriesDurations[i].minutes);
+    let SeriesTimeDurationInPercentage = (SeriesTimeDurationInMinutes * 100) / totalLifeTime;
```

Comment:
These should be `const`.

Reply:
And it's the standard to write variable names in `camelCase`:
`seriesTimeDurationInMinutes`

Reply:
There's a trick called "object destructuring", if you want to use it:
```js
const { days, hours, minutes } = seriesDurations[i]; // here we drag out the values from the object and assign them
// Now we can use them directly:
const seriesTimeDurationInMinutes = (days * 24 * 60) + (hours * 60) + (minutes);    
const seriesTimeDurationInPercentage = (seriesTimeDurationInMinutes * 100) / totalLifeTime;
```